src/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: gray;
+  margin: 0;
+}
+
+/* img {
+  margin: 0;
+}
+
+div {
+  margin: 0;
+} */
+
+.card_style {
+  height: 403px;
+  width: 200px;
+  padding: 15px;
+  border-radius: 5px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
+  border: 1px solid #f3f3f3;
+  box-shadow: 0 2px rgba(0, 0, 0, 0.07);  
+  /* background-color: yellow; */
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 15px;
+  align-self: center;
+}
+
+.card__info {
+  width: 100%;
+  /* height: 31px; */
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  line-height: 15px;
+  margin-top: 45px;
+}
+
+.card__code {
+  width: 100%;
+  /* height: 13px; */
+  font-size: 11px;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin-top: 6px;
+}
+
+.testimonials {
+  width: 100%;
+  display: flex;
+  /* flex-direction: row; */
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.stars {
+  display: flex;
+  justify-self: start;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  background: url("/images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__item:nth-child(-n+4) {
```

Comment:
add all classes from the previous task, so I can change the number at the end of a class name, and amount of stars will also change